composer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "momentohq/laravel-cache",
```

Comment:
This also needs a `repositories` section for `client-sdk-php` similar to the one suggested above.